<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>B590_imitator</class>
 <widget class="QWidget" name="B590_imitator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_addr"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SerialBox5" name="serial_widget" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_test">
     <property name="text">
      <string>test</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialBox5</class>
   <extends>QWidget</extends>
   <header>serialbox5.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
